pontoon/administration/templates/admin_project.html

```diff
@@ -1,5 +1,7 @@
 {% extends "landing.html" %}
 
+{% import 'widgets/locales.html' as locales %}
```

Comment:
Rename to `widgets/multiple_locale_selector.html` to be more precise and in unify with CSS/JS filenames.